config/locales/en.yml

```diff
@@ -101,3 +101,129 @@ en:
       your_answers: "Your answers"
       current_question: "Current question"
       change: "Change"
+  roadmap:
+    page_title: Roadmap - GOV.UK
+    hero:
+      description: We’re always working to improve GOV.UK to meet user needs. This is our current roadmap, which takes us to the end of March 2021. We plan to update it again in the new financial year.
+      heading: GOV.UK roadmap
+    numbers_section:
+      footnote: 'We can only count visits to the GOV.UK website if users give us permission to.'
+      heading_large: GOV.UK in numbers
+      heading_small: From April 2020 to January 2021
+      numbers_columns:
+        column_one_count: 235
+        column_one_text: million users
+        # the special characters are in place to keep this text on one line
+        column_two_text: "billion\u00A0page\u00A0views"
```

Comment:
This line is wrapped in quotes `"` but a few others are wrapped in `'`. Could this be made consistent throughout the file or is there a reason for this?

Reply:
There is a [reason](https://stackoverflow.com/questions/19706925/insert-html-special-characters-in-i18n-yml) for the double quotes in this particular case.